public/app/plugins/panel/stat/common.ts

```diff
@@ -107,3 +112,55 @@ export function addOrientationOption<T extends SingleStatBaseOptions>(
     defaultValue: VizOrientation.Auto,
   });
 }
+
+export function formatDisplayValuesWithCustomUnits(
+  fieldValues: FieldDisplay[],
+  config: PanelFieldConfig
+): FieldDisplay[] {
+  const { prefix, suffix } = config;
+
+  if (!prefix && !suffix) {
+    return fieldValues;
+  }
```

Comment:
Is this just a fail safe for usage in another panel if prefix / suffix aren't being checked? Seems a little redundant based on current usage / implementation 🤷‍♂️

Reply:
I was thinking a fail safe. As you mentioned, I have a condition in `StatPanel.tsx` that only invokes this functions if there is at least one custom value. But if that condition isn't implemented alongside this function elsewhere, this will catch the fully-nullish conditions. Now that you mention it, it's probably best to remove the condition in StatPanel and just have the passthrough in this function. Thanks!